source/reference/hyperlinks.txt

```diff
@@ -0,0 +1,81 @@
+.. _style-guide-hyperlinks:
+
+==========
+Hyperlinks
+==========
+
+You can link out to other sites, internal pages or sections using inline
```

Comment:
Does "internal" apply to both "pages" and "sections"? If it's supposed to, I wonder if there's a cleaner way to phrase this. E.g.: "You can link out to other sites or internal pages or sections using..."  It's not perfect, but I'm sure we can some up with something.

Reply:
It does! I'll clarify in my next commit.